cmd/crypto/kes.go

```diff
@@ -103,10 +108,28 @@ func NewKes(cfg KesConfig) (KMS, error) {
 		return nil, err
 	}
 	return &kesService{
-		client: kes.NewClient(cfg.Endpoint, &tls.Config{
-			Certificates: []tls.Certificate{cert},
-			RootCAs:      certPool,
-		}),
+		client: &kesClient{
+			addr: cfg.Endpoint,
+			httpClient: http.Client{
+				Transport: &http.Transport{
+					Proxy: http.ProxyFromEnvironment,
+					DialContext: (&net.Dialer{
+						Timeout:   30 * time.Second,
+						KeepAlive: 30 * time.Second,
+						DualStack: true,
+					}).DialContext,
+					ForceAttemptHTTP2:     true,
```

Comment:
You really want to do this, HTTP2 has quite a few bugs I think we should leave it out for now.

Reply:
For now it is not needed (for MinIO eventually never) but if / when there is a stream/encrypt and stream/decrypt KES API HTTP2 will be required to read and write to the same HTTP connection... However, I can remove for now.